LayoutTests/imported/w3c/web-platform-tests/css/cssom-view/cssom-getBoxQuads-001.html

```diff
@@ -33,19 +33,29 @@
   </div>
 
   <script>
+    let bb = document.getElementById("block-block")
+    let fb = document.getElementById("flex-block")
     test(function() {
-      let bb = document.getElementById("block-block");
-      assert_equals(bb.getBoxQuads({box: "border"})[0].getBounds().width,  20, "Block layout border box is expected width.");
-      assert_equals(bb.getBoxQuads({box: "margin"})[0].getBounds().width, 100, "Block layout margin box is expected width.");
+      assert_equals(bb.getBoxQuads({box: "border"})[0].getBounds().width, 20)
+    }, "Block layout border box is expected width.")
+    test(function() {
+      assert_equals(bb.getBoxQuads({box: "margin"})[0].getBounds().width, 100)
+    }, "Block layout margin box is expected width.")
 
-      // For containers that expand items to fill block-axis space, measure the box heights also.
-      let fb = document.getElementById("flex-block");
-      assert_equals(fb.getBoxQuads({box: "border"})[0].getBounds().width,  20, "Flex layout border box is expected width.");
-      assert_equals(fb.getBoxQuads({box: "margin"})[0].getBounds().width, 100, "Flex layout margin box is expected width.");
+    // For containers that expand items to fill block-axis space, measure the box heights also.
+    test(function() {
+      assert_equals(fb.getBoxQuads({box: "border"})[0].getBounds().width, 20)
+    }, "Flex layout border box is expected width.")
+    test(function() {
+      assert_equals(fb.getBoxQuads({box: "margin"})[0].getBounds().width, 100)
+    }, "Flex layout margin box is expected width.")
 
-      assert_equals(fb.getBoxQuads({box: "border"})[0].getBounds().height, 10, "Flex layout border box is expected height.");
-      assert_equals(fb.getBoxQuads({box: "margin"})[0].getBounds().height, 50, "Flex layout margin box is expected height.");
```

Comment:
Fwiw, you can also use the `single_test` setup, which would let you avoid calling test() everytime: https://web-platform-tests.org/writing-tests/testharness-api.html#single-page-tests

Same for the other test.

```
<title>Flex layout margin box is expected height</title>
<script>
    setup({ single_test: true });
    // For containers that expand items to fill block-axis space, measure the box heights also.
    let fb = document.getElementById("flex-block");
    assert_equals(fb.getBoxQuads({box: "border"})[0].getBounds().width,  20, "Flex layout border box is expected width.");
    assert_equals(fb.getBoxQuads({box: "margin"})[0].getBounds().width, 100, "Flex layout margin box is expected width.");

    assert_equals(fb.getBoxQuads({box: "border"})[0].getBounds().height, 10, "Flex layout border box is expected height.");
    assert_equals(fb.getBoxQuads({box: "margin"})[0].getBounds().height, 50, "Flex layout margin box is expected height.");
    done();
</script>
```

Reply:
Sure, that seems better for adapting these tests.

Reply:
Actually I just realized `single_test` will probably not use the output you want :( 

I'm fine with the current patch as it is too fwiw.